samples/dotnet/kernel-syntax-examples/Example02_Pipeline.cs

```diff
@@ -23,8 +23,8 @@ public static async Task RunAsync()
         var text = kernel.ImportSkill(new TextSkill());
 
         SKContext result = await kernel.RunAsync("    i n f i n i t e     s p a c e     ",
-            text["LStrip"],
```

Comment:
These values come from `\samples\dotnet\kernel-syntax-examples\Skills\TextSkill.cs` which contain the `LStrip` and `RStrip` semantic functions. `TrimStart` and `TrimEnd` come from the TextSkill in the kernel's core skills. The build breaks with this current change.